ocs_ci/deployment/deployment.py

```
@@ -3171,6 +3165,27 @@ def muliclusterhub_running(self):
+    ``wildcardPolicy: Subdomain`` so that all ``*.apps.<client>.*``
+    traffic is forwarded to the hosted cluster's own router.
+    Without this patch the route is rejected with
+    ``wildcard routes are not allowed``.
+    """
+    logger.info("Enabling wildcard routes on default IngressController")
+    path = "/spec/routeAdmission"
+    value = '{wildcardPolicy: "WildcardsAllowed"}'
+    params = f"""[{{"op": "add", "path": "{path}", "value": {value}}}]"""
+    patch_cmd = (
+        f"patch {constants.INGRESSCONTROLLER} -n {constants.OPENSHIFT_INGRESS_OPERATOR_NAMESPACE} "
+        + f"default --type json -p '{params}'"
```

Ruff (0.15.21)

[warning] Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)